src/Components/Clock.js

```diff
@@ -0,0 +1,14 @@
+import { Row } from 'react-bootstrap';
+
+/** function generating a timer clock
+ * @returns {Node} representing the component
+ */
+function Clock() {
+  return (
+    <Row xs={1} className="justify-content-center m-auto my-0">
+      <p className="text-center fw-bold my-0" style={{ fontSize: '15vw' }}>23:12</p>
```

Comment:
### Nitpick

let's have a variable called `clockStyle` and use it to store our styles rather to be hardcoding styles right in there.

```javascript
const clockStyle = { fontSize: '15vw' }

 <p className="text-center fw-bold my-0" style={clockStyle}>23:12</p>

```